### PR TITLE
Cosmetic change

### PR DESCRIPTION
**Pull request by krvikash**
## Description

First PR in trino. Cosmetic Change for delta-lake, iceberg, and hive tests

## Non-technical explanation
NA

## Release notes

(x) This is not user-visible and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/AbstractTestParquetReader.java [hunk 1]
```diff
@@ -1425,7 +1425,7 @@ public void testNewAvroArray()
 
     /**
      * Test reading various arrays schemas compatible with spec
-     * https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#lists
+     * <a href="https://github.com/apache/parquet-format/blob/master/LogicalTypes.md#lists">here</a>
```
Comment (findinpath) [inline plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/AbstractTestParquetReader.java:1428]: Not necessary

File: plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java [hunk 2]
```diff
@@ -2263,7 +2263,7 @@ public final Object[][] bucketedUnsupportedTypes()
     }
 
     /**
-     * Regression test for https://github.com/trinodb/trino/issues/5295
+     * Regression test for <a href="https://github.com/trinodb/trino/issues/5295">trino#5295</a>
```
Comment (findinpath) [inline plugin/trino-hive/src/test/java/io/trino/plugin/hive/BaseHiveConnectorTest.java:2266]: Not necessary

File: plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/AbstractTestParquetReader.java (hunk 1 above)
Comment (findepi) [reply plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/AbstractTestParquetReader.java:1428]: I agree that `<a href...` is the correct way to embed links in javadoc.
Note that tests'  javadocs are read within IDE and then plain link (without `<a>`) is simply more readable.
Javadocs for production code, especially for SPI, may be published as web pages and they need to be properly formed. They should use `<a>` tags for links.

Comment (krvikash) [reply plugin/trino-hive/src/test/java/io/trino/plugin/hive/parquet/AbstractTestParquetReader.java]: reverted the java doc changes.

**Comment by findinpath** [comment]
@krvikash  please follow the guidelines from https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md#commits-and-pull-requests for the commit messages.

**Comment by krvikash** [comment]
Closing this PR, As there are some unwanted commit gets included into this.

**Pull request closed without merging**